Dockerfile

```diff
@@ -14,29 +14,30 @@ RUN apt-get update && \
       procps && \
     pip install --upgrade pip
 
-COPY CHANGES.rst README.rst setup.py /code/
-COPY reana_server/version.py /code/reana_server/
+# Copy cluster component source code
 WORKDIR /code
-RUN pip install --no-cache-dir requirements-builder && \
-    requirements-builder -l pypi setup.py | pip install --no-cache-dir -r /dev/stdin && \
-    pip uninstall -y requirements-builder
-
 COPY . /code
 
-# Debug off by default
+# Install dependencies
+RUN pip install -r requirements.txt
```

Comment:
Note that this was not optimal WRT docker layer caching: changing anything in the source code would trigger reinstalling of dependencies. I'll force-push an update here and elsewhere.